### PR TITLE
replace functions that were removed from the sdk

### PR DESCRIPTION
**Pull request by jarry-xiao**
Added some additional units renaming to the orderbook class. No more use of the ambiguous `mult` keyword in variables

### CODE REVIEW

File: rust/examples/src/bin/sample.rs [hunk 1]
```diff
@@ -106,7 +106,9 @@ async fn main() -> anyhow::Result<()> {
             market_pubkey, header.quote_params.mint_key, header.base_params.mint_key
         );
 
-        if header.base_params.mint_key == devnet_token_faucet::get_mint_address("SOL") {
+        if header.base_params.mint_key == devnet_token_faucet::get_mint_address("SOL")
+            || header.base_params.mint_key == spl_token::native_mint::id()
```
Comment (eugene-chen) [inline rust/examples/src/bin/sample.rs:110]: this is `So1111...`?

File: rust/phoenix-sdk/Cargo.toml [hunk 2]
```diff
@@ -29,5 +29,5 @@ rust_decimal = { workspace = true }
 rust_decimal_macros = { workspace = true }
 bytemuck = { workspace = true }
 itertools = "0.10.5"
-phoenix-sdk-core = { version = "0.3.5", path = "../phoenix-sdk-core" }
+phoenix-sdk-core = { version = "0.3.6", path = "../phoenix-sdk-core" }
```
Comment (eugene-chen) [inline rust/phoenix-sdk/Cargo.toml:32]: newline at end of file please

File: rust/phoenix-sdk/src/sdk_client.rs [hunk 3]
```diff
@@ -280,23 +280,31 @@ impl SDKClient {
             .await
             .unwrap_or_default();
         let default = Orderbook::<FIFOOrderId, PhoenixOrder> {
```
Comment (eugene-chen) [inline rust/phoenix-sdk/src/sdk_client.rs:282]: is default a keyword? github is coloring it differently. if it is a reserved term let's rename

File: rust/phoenix-sdk/src/sdk_client.rs [hunk 4]
```diff
@@ -280,23 +280,31 @@ impl SDKClient {
...
             asks: BTreeMap::new(),
         };
         if market_account_data.is_empty() {
             return Ok(default);
         }
         let (header_bytes, bytes) = market_account_data.split_at(size_of::<MarketHeader>());
-        let base_lots_multiplier = self.base_lots_to_base_units_multiplier(market_key)?;
-        let ticks_multiplier = self.ticks_to_float_price_multiplier(market_key)?;
+        let meta = self.get_market_metadata_from_cache(market_key)?;
+        let raw_base_units_per_base_lot =
```
Comment (eugene-chen) [inline rust/phoenix-sdk/src/sdk_client.rs:293]: why isn't this number instead `base_lots_per_raw_base_unit`? or is that also not guaranteed to be an integer?

**Review by eugene-chen — APPROVED** [review]

File: rust/examples/src/bin/sample.rs (hunk 1 above)
Comment (jarry-xiao) [reply rust/examples/src/bin/sample.rs:110]: yes

File: rust/phoenix-sdk/src/sdk_client.rs (hunk 3 above)
Comment (jarry-xiao) [reply rust/phoenix-sdk/src/sdk_client.rs:282]: I do not think that default is a keyword in rust

File: rust/phoenix-sdk/src/sdk_client.rs (hunk 4 above)
Comment (jarry-xiao) [reply rust/phoenix-sdk/src/sdk_client.rs:293]: This is the number that's multiplied to base lots to return a human readable price

Comment (jarry-xiao) [reply rust/phoenix-sdk/src/sdk_client.rs:293]: In the majority of cases it's going to be < 1

**Pull request merged**